python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -418,10 +418,10 @@ def _get_unique_target_path(self, *, context: Optional[OpExecutionContext]) -> s
         Returns:
             str: A unique target path for the dbt CLI invocation.
         """
-        current_unix_timestamp = str(int(time.time()))
-        path = current_unix_timestamp
+        current_unix_timestamp_ms = str(int(time.time() * 1e3))
```

Comment:
if we want to be extra safe, we could always add a few random letters after this

Reply:
yeah maybe i'll just add a UUID

Reply:
or just a uuid()

Reply:
~or just a uuid~ whoops you just said that